.devcontainer/setup.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash 
+set -euo pipefail
+
+# Go to the workspace directory: 
+if [ -d "py-launch-blueprint" ]; then
+  echo "In container: switching to workspace dir"
+  cd py-launch-blueprint
+else
+  echo "Not in container: skipping cd to /workspaces"
+fi
+
+# Create virtual environment if not exists
+if [ ! -d ".venv" ]; then
+  echo "Creating virtual environment..."
+  python3 -m venv .venv
+fi
+
+# Activate virtual environment
+if [ -f ".venv/bin/activate" ]; then
+  echo "Activating .venv (Linux/macOS)..."
+  source .venv/bin/activate
+elif [ -f ".venv/Scripts/activate" ]; then
+  echo "Activating .venv (Windows)..."
+  source .venv/Scripts/activate
+else
+  echo "No virtual environment found. Skipping activation."
+fi
+
+# Install dependencies
```

Comment:
Dependencies should use the existing just commands. Again, we're deviating from the patterns that we're using in the project. If there's a reason that this can't or shouldn't, I would like to have a proposal of why. Otherwise, we should use the stator commands in this environment.